token-metadata/js/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@solana/spl-token-metadata",
     "description": "SPL Token Metadata Interface JS API",
-    "version": "0.1.0",
```

Comment:
How do you all manage versions? Do you strictly follow semver? Cause then i think it should be 1.0.0

Reply:
I'll do the version bump separately!